### ISSUE TITLE
ValueError: Chunk too big in async stream.

### ISSUE THREAD

**Issue by amrit073** [cohere-ai/cohere-python#402]
I was trying to use the async stream to summarize a document, which was fairly long.
I got this error:

```
  File "./minimal.py", line 28, in get_summary_stream_cohere_client
    async for token in streaming_chat:
  File "./venv/lib/python3.11/site-packages/cohere/responses/chat.py", line 370, in __aiter__
    async for line in self.response.content:
  File "./venv/lib/python3.11/site-packages/aiohttp/streams.py", line 35, in __anext__
    rv = await self.read_func()
         ^^^^^^^^^^^^^^^^^^^^^^
  File "./venv/lib/python3.11/site-packages/aiohttp/streams.py", line 311, in readline
    return await self.readuntil()
           ^^^^^^^^^^^^^^^^^^^^^^
  File "./venv/lib/python3.11/site-packages/aiohttp/streams.py", line 338, in readuntil
    raise ValueError("Chunk too big")
```

It turns out the API responds with the whole chat_history at the end of the stream, which is too big for aiohttp stream.
` return_chat_history=False` is not being respected.

Minimal reproduction code:

```
import json

import cohere

import config

SYS_PROMPT = """
Provide a concise one-page summary of the document provided to you.
""".strip()


async def get_summary_stream_cohere_client(text: str):
    async with cohere.AsyncClient(config.COHERE_API_KEY) as aio_co:
        streaming_chat: cohere.StreamingChat = await aio_co.chat(  # type:ignore
            message=f"""
                <document>
                {text}
                </document>
                {SYS_PROMPT}
                Summary:
                """.strip(),
            max_tokens=3000,
            model="command-r",
            stream=True,
  
        )
        async for token in streaming_chat:
            if hasattr(token, "text") and token.text is not None:
                print(token.text)


if __name__ == "__main__":
    import asyncio

    # https://0x0.st/HFN3.json
    with open("./this.json", "r") as f:
        file_content = json.load(f)
        text = file_content["text"]
        asyncio.run(get_summary_stream_cohere_client(text))
```

**Comment by harry-cohere** [comment]
Thanks for reporting @amrit073 !

We think we've fixed the issue in #409 - please could you try the most recent v4 client (4.57)?

Additionally, we checked that this is working well in the v5 SDK (currently in alpha).

Thanks!

**Comment by amrit073** [comment]
Thanks!
It's now working as expected.
